JDBC/OAuthSampleApp/src/main/java/OAuthSampleApp.java

```diff
@@ -88,12 +279,28 @@ public Integer call() throws Exception {
             System.out.println(" Check the login credentials and try again.");
         } catch (SQLException e) {
             e.printStackTrace();
+        }finally {
+		//TearDown();
         }
         return 0;
     }
-
+    
+    // load configuration properties
+    public static void loadProperties()
+    {
+	    prop = new Properties();
+		try (InputStream input = new FileInputStream("src/main/java/config.properties")) {
+	        // load a properties file
+	        prop.load(input);
+	    } catch (Exception ex) {
+	        ex.printStackTrace();
+	    }
+	}
+    
     public static void main(String[] args) {
-        int exitCode = new CommandLine(new OAuthSampleApp()).execute(args);
+    	
+    	loadProperties();
+    	int exitCode = new CommandLine(new OAuthSampleApp()).execute(args);
         System.exit(exitCode);
     }
 
```

Comment:
Fix the spacing

Reply:
Fixed spacing. git viewer has to be adjusted with tab stop as 4 to view the file with proper indentation.